core/storage/feedback/gae_models_test.py

```diff
@@ -490,6 +585,38 @@ def test_export_data_nonexistent_case(self) -> None:
             self.USER_ID_B)
         self.assertEqual({}, user_data)
 
+    def test_apply_deletion_policy(self) -> None:
```

Comment:
Same thing, for this and others -- don't just write "test {{method name}}", write "test {{behaviour}}". See guidance [here](https://github.com/oppia/oppia/wiki/Backend-tests#guidelines-for-writing-good-tests).

Reply:
Done